build.d/10-ssh-key

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python
+
+import logging
+import os
+import subprocess
+
+
+SSH_DIR = '/opt/odoo/custom/ssh.d'
+CONFIG_PATH = os.path.join(SSH_DIR, 'config')
+
+
+if not os.path.isfile(CONFIG_PATH):
+    logging.info('No SSH configuration found')
+    exit(0)
+
+
+HOMES = {
+    'root': '/root',
+    'odoo': '/opt/odoo',
```

Comment:
No need for these, just `os.path.expanduser("~root")` as needed.

OTOH, as said, no need for adding deploy keys for odoo user.